src/style.css

```
@@ -1 +1,68 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  padding-left: 50px;
+  padding-right: 50px;
```

use shorthand property
```suggestion
  padding: 0 50px;
```